File: P0_send_client1_mqtt.rb
#!/usr/bin/env ruby

require "mqtt"

c = MQTT::Client.connect(
  remote_host: "127.0.0.1",
  remote_port: 1883,
  username: "guest",
  password: "guest")

loop do
  m = "Message @ #{Time.now.to_i}"
  c.publish("chat.client1ID", m)
  puts "=> Published #{m}"
  sleep 0.2
end

File: P0_send_client1_mqtt.py
import time
from datetime import datetime
import paho.mqtt.client as mqtt

client = mqtt.Client()
client.username_pw_set("test", "test")
client.connect("127.0.0.1")

while(1):
  msg = "Hello client1 " + str(datetime.now())
  client.publish("chat.client1ID", msg)
  print "publish", msg
  time.sleep(1)
